src/test/datascience/jupyter/kernels/installationPrompts.vscode.test.ts

```diff
@@ -91,54 +96,68 @@ suite('DataScience Install IPyKernel (slow) (install)', function () {
     });
 
     test('Ensure prompt is displayed when ipykernel module is not found and it gets installed', async () => {
-        const installed = createDeferred();
+        // Do this for both kernels
+        await Promise.all(
+            ['.venvnokernel', '.venvnoreg'].map(async (kName) => {
```

Comment:
Doesn't this mean we're running both together? I.e. opening two notebooks side by side & running them for both kernels?
Did you instead mean to put this for loop outside the test & run each test seprately.

Reply:
Yeah that's what I intended. Not in parallel

Reply:
Test isn't passing. Can't get Peng's command to work. Going to disable for now.